### ISSUE TITLE
cannot import name 'anchor_generator_pb2' from 'object_detection.protos' 

### ISSUE THREAD

**Issue by N04H8** [nicknochnack/TFODCourse#98]
When running a verification script this error outputs:  ImportError: cannot import name 'anchor_generator_pb2' from 'object_detection.protos' 
I have been stuck at this error for many days and all the solutions haven't worked.
Any help is greatly appreciated.

**Comment by Dex247** [comment]
Same as me. Were you able to get a solution? Edit................. I got it fixed. I changed the location of some files from their original folders to the script directory. That worked for me.

**Comment by N04H8** [comment]
Nope, still stuck at the same error. Let me know if you figure anything out.

**Comment by Dex247** [comment]
It seems we are both at the same spot. add me up on whatsapp or telegram so
we can discuss how best we can tackle the issue

**Comment by Shreyas2309** [comment]
I am facing the same error

**Comment by Devilshree123** [comment]
I am facing the same error
